src/index.js

```diff
@@ -1,6 +1,12 @@
 // Polyfill `Promise`.
 window.Promise = window.Promise || require('promise-polyfill');
 
+// Custom Elements v1 polyfill
+if (!window.customElements) {
+  const customElementsPolyfill = require('@webreflection/custom-elements');
+  customElementsPolyfill(window || global);
```

Comment:
you check for `!window.customElements` without checking if `window` exists at all, so this will never pass through the `||`, or better, code will fail anyway if `window` is not around. I suggest to either remove that `|| global`, as never reached, and maybe use `self` instead of window for Worker cases, or even better: `globalThis`.

That being said, you could use [the proper, maintained, polyfill](https://github.com/ungap/custom-elements#readme) instead.